### PR TITLE
CONCF-173 scroll to link fix for global nav

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by bognix** [comment]
ping @hakubo @vforge

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationScrollToLink.js [hunk 1]
```diff
@@ -37,6 +25,37 @@ require(
 		};
 
 
-		// bind to DOMReady
-		$(initScrollToLink);
+		$(function() {
+			var $win = $(win),
+				SPACING_BELOW_GLOBAL_NAV = 10,
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationScrollToLink.js]: I think our guide lines says do NOT_USE_UPPERCASE :)

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationScrollToLink.js [hunk 2]
```diff
@@ -37,6 +25,37 @@ require(
...
+				globalNavigationHeight,
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationScrollToLink.js]: why declare and initialise on 2 separate lines?

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationScrollToLink.js [hunk 3]
```diff
@@ -37,6 +25,37 @@ require(
...
+				});
+			}
+
+			// offset is negative - we want scroll BEFORE element's top offset
+			// also scroll a bit, so element won't be sticked to GlobalNavigation
+			offset = -( globalNavigationHeight + SPACING_BELOW_GLOBAL_NAV);
+
+			$win.on('hashchange', hashChangeHandler);
+
+			$win.on('load', function() {
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationScrollToLink.js]: This can be chained

**Comment by vforge** [comment]
LGTM

**Pull request merged**